library/Vanilla/InternalRequest.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * @author Todd Burry <todd@vanillaforums.com>
+ * @copyright 2009-2017 Vanilla Forums Inc.
+ * @license GPLv2
+ */
+
+namespace Vanilla;
+
+use Garden\Web\RequestInterface;
+
+class InternalRequest implements RequestInterface {
```

Comment:
This shouldn't be here, should it? I'm not seeing any references to this class in the updates.